int32[] JointIDs
int32 ActuatorMode